bff-service/app.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express';
+import cors from 'cors';
+import axios from 'axios';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+const port = process.env.PORT || 4000;
+export const headers = {
+  'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+  'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+};
+
+app.use(cors());
+app.use(express.json());
+
+let cache = {};
+
+setInterval(() => {
+  cache = {};
+}, 10000);
```

Comment:
Seems like you need to put 120000 here (by task description).

Reply:
Common, it is not so important